README.md

```diff
@@ -1 +1,145 @@
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖  Portfolio-project <a name=" Portfolio-project">My Portfolio Project</a>
+
+
+
+**Portfolio-project** This is my portfolio project. 
+Portfolio-project is a mobile and desktop responsive page.
+
+## 🛠 Built With HTML <a name="For Text"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+HTML5 and CSS which are the latest versions were used in building this project.
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">JAVASCRIPT</a></li>
+  </ul>
+</details>
+
+
+
+
+### Key Features <a name="key-features"></a>
+
+- **[key_feature_1]** Portfolio-project built with HTML and CSS.
+- **[key_feature_2]** Responsive mobile design portfolio.
+
+
+- **feature_HOME-PAGE**
+
+## 🚀 Live Demo <a name="Live Demo"></a>
+
+
+)
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+Get your pc.
+Clone this project to your local machine.
+
+To get a local copy up and running, follow these steps.
+
+
+
+### Prerequisites
+Basic knownledge of computer.
+
+In order to run this project you need:
+Live server.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+  cd Microverse_portfolio
+  git clone https://github.com/adeola003/Microverse_portfolio.git
+```
+
+## 👥 Authors <a name="Abdel-Khafid SALAOU"></a>
+
+
+👤 **Abdel-Khafid**
+
+- GitHub: [@githubhandle](https://github.com/adeola003)
+- Twitter: [@twitterhandle](https://twitter.com/khadaf6)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/abdel-khafid-salaou-a18396161/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+
+- **About-page**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+
+If you like this project you can follow my github or star it
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+
+All my 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] This section should not be added as an image, kindly use the Microverse template correctly, the author's social media should be clickable links as well as the MIT license.

Reply:
@algerina  Good morning sir. thank you for your time and for helping me to get better at this. I followed your recommendations on the project and I'm about to submit a new review request.
Best regards.